jocke1995/StortSpelProjekt
# Encryption of resources.

To test the encryption,

include the cryptor class:
#include "Misc/Cryptor.h"

Add this code to main (preferrably before any loads) and adjust filepaths to a file of your own choosing:
    unsigned int key = time(NULL);
    Cryptor::Encrypt(key, "INSERT SOURCE FILEPATH", "INSERT TARGET FILEPATH");
    Cryptor::Decrypt(key, "INSERT SOURCE FILEPATH", "INSERT TARGET FILEPATH");

Edit:
Note that the filepath is relative to the project folder the project you are building from.
